coda/coda_mdstore/urls.py

```diff
@@ -31,42 +32,29 @@
     url(r'^about/$', views.about, name='about'),
     url(r'^robots.txt$', views.shooRobot, name='robots'),
     url(r'^feed/$', views.AtomSiteNewsFeed(), name='feed'),
-    url(r'^$', views.index, name='index'),
-]
-
-
-urlpatterns += patterns(
-    'django.contrib.sitemaps.views',
-    (
-        r'^resourceindex\.xml$',
-        'index',
+    url(r'^resourceindex\.xml$',
+        sitemap_views.index,
         {
             'sitemaps': sitemaps,
             'template_name': 'mdstore/resourceindex.xml'
         },
-    ),
-    (
-        r'^resourcelist-(?P<section>.+)\.xml$',
-        'sitemap',
+        ),
```

Comment:
Very minor style thing: I would pull this `),` back one indent and do the same for the next couple `url` calls as well.